games/space-invaders/Space Invaders [David Winter].json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Space Invaders",
+  "authors": [
+    {
+      "id": "david-winter",
+      "name": "David Winter"
+    }
+  ],
+  "release": "1978",
+  "category": "GAME",
+  "description": "The well known game. Destroy the invaders with your ship.",
+  "images": [
+    "./Space Invaders [David Winter] - main screen.png",
+    "./Space Invaders [David Winter] - game play.png"
+  ],
+  "interpreter": {
+    "id": "chip8-david-winter",
+    "name": "CHIP8 by David Winter"
+  },
+  "binaries": {
+    "names": [
+      { "value": "Space Invaders [David Winter] - 0.9.1" },
+      { "value": "Space Invaders [David Winter]" },
+
+      { "value": "Space Invaders [David Winter] - 0.9.0" },
+      { "value": "Space Invaders [David Winter] (alt)" },
+      { "value": "INVADERS" }
+    ],
+    "extensions": [
+      "c8",
+      "ch8",
+      ""
+    ],
+    "hashes": [
+      {
+        "type": "SHA256",
+        "value": "00bb7001de52b562d98357d33375406c6d0308466d7d95c2580537d2451a75cd"
+      },
+      {
+        "type": "SHA256",
+        "value": "2d0e1fa53216b297e74041d4fb766f42327a42893e83bb4ec931a9dff5c2dd10"
+      }
+    ]
+  },
+  "sources": {
+    "names": [
+      {
+        "type": "DISASSEMBLED",
+        "value": "Space Invaders [David Winter] - 0.9.0"
+      },
+      {
+        "type": "DISASSEMBLED",
+        "value": "Space Invaders [David Winter] - 0.9.1"
+      }
+    ],
```

Comment:
This section lists the source files of the app. For apps created for octojam where source is available this will be helpful. In other cases a disassembly could be provided (I didn't include the files from octo since this repo is metadata only).

Again, companion file case.
`name.type` would be `DISASSEMBLED` or `ORIGINAL`

Reply:
I replaced the type with comment, we might get many different "heterogeneous" types of sources so it's better to describe them in free text form.